script/gen_requirements_all.py

```diff
@@ -154,7 +154,10 @@ def gather_recursive_requirements(domain, seen=None):
 
 def comment_requirement(req):
     """Comment out requirement. Some don't install on all systems."""
-    return any(ign.lower() in req.lower() for ign in COMMENT_REQUIREMENTS)
+    idx=req.find("==")
+    if idx >= 0:
+      return any(ign.lower() == req[0:idx].lower() for ign in COMMENT_REQUIREMENTS)
+    return any(ign.lower() == req.lower() for ign in COMMENT_REQUIREMENTS)
```

Comment:
Why not use:

```suggestion
    return any(f"{ign.lower()}==" in req.lower() for ign in COMMENT_REQUIREMENTS)
```

Reply:
I'm not actually sure if package manifests are allowed to use `module>=version` (although my code still doesn't handle that case correctly either).  If we can depend on it always being `module==version`, then yes that is 100% better.

Reply:
We only allow `==`

Reply:
Perfect. Accepting the suggestion now.